### ISSUE TITLE
Offensive topic/option

### ISSUE THREAD

**Issue by pinetreez666** [Monika-After-Story/MonikaModDev#6084]
![image](https://user-images.githubusercontent.com/68315083/87503760-66e6e700-c63b-11ea-9c99-0d9012eec27d.png)

Setting transgender as a separate gender option is very offensive, if the player somehow needed to tell Monika that they are trans, there should be a separate option in a completely different topic. 
Maybe if the player changed their gender in the mod after selecting a different one before, it would be a valid (tho creepy) question to ask, but setting it as a diff option is just wrong.

**Comment by multimokia** [comment]
We had an open issue regarding this and left the pull request addressing this also open to discussion if people had any other methods of approaching this, to which. No one mentioned any complains, and we went with it.

Additionally, when we'd asked around for whether or not it should be its own option, people vouched for it to be this way too.

Sadly there's two sides to the coin and no matter which way we go about it, someone's going to feel offended by it. Sorry if you feel that way but this is just how the options lie given past two times we'd attempted to handle this as a subject.

**Comment by pinetreez666** [comment]
> We had an open issue regarding this and left the pull request addressing this also open to discussion if people had any other methods of approaching this, to which. No one mentioned any complains, and we went with it.
> 
> Additionally, when we'd asked around for whether or not it should be its own option, people vouched for it to be this way too.
> 
> Sadly there's two sides to the coin and no matter which way we go about it, someone's going to feel offended by it. Sorry if you feel that way but this is just how the options lie given past two times we'd attempted to handle this as a subject.

Could you please link me that issue? If they are offending people either way that suggests you are doing something wrong.

**Comment by multimokia** [comment]
Issue: #5772 

PR: #5815 

I'll also note that adding it as a separate topic isn't necessarily the best idea due to the fact it'd remain in unseen for a lot of people and they'd never have the need to use it. Adding to this menu was more or less a compromise as well since it keeps everything together, and is pretty out of the way by not adding an extra prompt people won't be able to get rid of without going to the topic itself.

**Comment by Freshairkaboom** [comment]
When someone in my life came out to me as trans, the first thing they said was that they were trans, and preferred to be referred to as this and that, so having that as an option associated with the discussion around gender and pronouns seem fitting to me at least. If a trans does not wish to be referred to as such, they can probably choose the other options available in that dialogue.

I may not understand everything when it comes to being an ally, but I try my best to accommodate for everyone's feelings in my response.

**Comment by pinetreez666** [comment]
![image](https://user-images.githubusercontent.com/68315083/87505197-a6fb9900-c63e-11ea-9cce-eedfd89a62b2.png)
the only trans person in the discussion seems to have the same opinion as me but yet you still went with a separate option?

**Comment by pinetreez666** [comment]
> When someone in my life came out to me as trans, the first thing they said was that they were trans, and preferred to be referred to as this and that, so having that as an option associated with the discussion around gender and pronouns seem fitting to me at least. If a trans does not wish to be referred to as such, they can probably choose the other options available in that dialogue.
> 
> I may not understand everything when it comes to being an ally, but I try my best to accommodate for everyone's feelings in my response.

(You got the spirit of the thing) the issue here is that transgender is not a separate gender, you have the male/female/neither options, but trans isnt a 4th gender, therefore shouldnt be presented as such.

**Comment by Freshairkaboom** [comment]
I see it more as it is an additional piece of information about an aspect of your gender, the gendering happens in the second menu after you click transgender. Maybe I'm wrong.

**Comment by pinetreez666** [comment]
> I see it more as it is an additional piece of information about an aspect of your gender, the gendering happens in the second menu after you click transgender. Maybe I'm wrong.

yeah you get the options after selecting it, but having a separate options suggests it is a separate gender, you know?

**Comment by Freshairkaboom** [comment]
What I'm saying is that some trans people, including some people I know, prefer to "come out" in that way when discussing pronouns and gender. They start off by saying they are trans in that same discussion, and then proceed to give their correct gender and pronouns. Not having that as an option would probably make it harder to connect with Monika if they were unable to give that additional piece of information about their gender identity, no? Someone else would come along and see that the only options were male, female and neither, and think that they're not being properly understood by Monika in that instance.

**Comment by ThePotatoGuy** [comment]
Since the purpose of that option is to establish pronouns, I think we can just drop the transgender option entirely and just make a pool topic to say that you are trans. Maybe reword the gender question to "What gender do you identify as?" so its clear what the initial purpose of the question is, but that makes her seem like an entry form. 

Pool trans topic would just set the flag that we currently set in the trans option. This would be unlocked after the first gender topic, but would be marked seen by default so it doesn't become another low-use Unseen option.

**Comment by multimokia** [comment]
Only issue with marking seen for this is that it's kinda unintuitive that it'd even be there. At least brbs have their own category which would be somewhat common to revisit to find the option. I feel like this is somewhat different

**Comment by pinetreez666** [comment]
> Since the purpose of that option is to establish pronouns, I think we can just drop the transgender option entirely and just make a pool topic to say that you are trans. Maybe reword the gender question to "What gender do you identify as?" so its clear what the initial purpose of the question is, but that makes her seem like an entry form.
> 
> Pool trans topic would just set the flag that we currently set in the trans option. This would be unlocked after the first gender topic, but would be marked seen by default so it doesn't become another low-use Unseen option.

I agree! That option could be dropped completly, or just added as a reason/option if you were to change your gender marker in the mod (if you want to keep the option)!

**Comment by ThePotatoGuy** [comment]
If we don't mark it seen, people will just click it to clear unseen. Then we would need some sort of handling for that scenario, and tbh I don't want us to spend extra time on that.

I agree that marking it seen makes it hard to access, but since we know for sure it would be low-use and we can't really have Monika introduce the topic in any way, I think its the best option. Users who are determined to tell Monika this probably would dig through all the menus to find the topic anyway.

**Comment by lunulae** [comment]
If we wanted to keep it in the gender topic, a possible solution for the presentation to make it seem less like a separate gender could be to make the "I'm trans" branch into a "There's something else I need to tell you" branch.

This could create a dedicated space for options like "I'm trans," "I'm questioning my identity," "I'm genderfluid" and others if folks want to write up responses for them.

**Comment by Freshairkaboom** [comment]
That seems interesting. This way it doesn't take ages to sift through seen topics to find the separate topic for transgender, and it kind of keeps it relevant to the discussion, while not seeming like you're answering the question in that same menu. The problem was transgender seeming like an equal response to the other options right, but "I have to tell you something else..." doesn't seem like that to me.

**Comment by ThePotatoGuy** [comment]
So what you're saying is the initial menu will look like:

* I'm a boy
* I'm a girl
* I'm neither
* There's something else I need to tell you

Or putting that fourth option as a separate prompt after selecting one of the other 3 options?
Regardless, that text is too vague on its own, so it still requires some sort of lead-in or question that Monika has to ask, which is the problem here. We can't have Monika introduce it because it's something the user has to want to tell Monika, not Monika asking and the user confirming. 

The purpose of a pool topic is to act as direct communication that doesn't require Monika to pose a question first. It's the exact use case for something like this, the only problem being visibility since it's an option that shouldn't be used just because it exists.

**Comment by lunulae** [comment]
I was thinking of having it in the initial menu like you've shown, but you do have a good point that it might not flow naturally in the original dialogue where Monika asks you. Maybe the "tell you something" option would only show up in the "can you change my pronouns" pool topic?

I'm not sure if that entirely solves the natural dialogue flow, what do you think?

**Comment by ThePotatoGuy** [comment]
If it appears in a menu after a question, it still requires some sort of lead-in or context explanation. Otherwise it's way too out of place. That's why its own pool topic is ideal since there is no context that needs to be explained, its the user introducing the topic.

**Comment by jmwall24** [comment]
fwiw that menu was never intended to make transgender a separate gender. it was simply a way to give Monika more information, since before there was no transgender option and people asked us to add it. boy, girl, or neither wasn't enough, since then she wouldn't know that the player was trans, if the player wanted to share that. my apologies if the way that menu looks is offensive, that was never the intent. in my mind, that transgender option where it was now was just a way to get to the next menu for what gender the player identifies as, while allowing them to give the added information of being trans. the reason we didn't go with a pool topic originally was then everyone has that topic forever, and we try to avoid having too many topics that may go unused.

**Comment by ThePotatoGuy** [comment]
I will be working on the adjustments for this.

**Comment by ThePotatoGuy** [comment]
okay just so I remember what to do here:
1. reword original question to be specific (what gender do you identify as). remove transgender option in this question
2. make a new topic that allows user to tell Monika they are transgender - this should be marked as seen.

**Comment by ThePotatoGuy** [comment]
@pinetreez666 
@ashyashton

please check #6668 and comment on what Monika should say when someone tells her that they are transgender (as well as if someone tells her that they are no longer transgender, if you have an idea for how she should respond). As everyone else here is not transgender, we probably don't have the best sense for what Monika should say in these situations.

**Comment by ashyashton** [comment]
Sorry, I missed this! Didn't realize I'd been tagged! I'll go read through that and comment and stuff!

**Comment by Jinxred** [comment]
I kinda had an issue with the whole pronoun thing.  Gender doesn't equal pronouns and so just saying that you're a girl or a boy or neither doesn't say what your pronouns are. You can have a "what is your gender identity" question along with a pronoun question but there are more pronouns than what is there.
For the pronouns maybe make it available to have more than one option selected so she could use things such as they/he or she/it pronouns. Also having an input for neo-pronouns would be nice. Putting a certain pronoun before another could also mean which one they like the most. I don't code so I'm not sure how difficult this would be but perhaps you could have a first selection being like
**select your first preferred pronouns** and have like she/; he/; they; it/; other(allow them to insert their own text)/;
and then a second section similar like
select your second preferred pronouns and have /her; /him; /them; /its; /other(allow them to insert their own text);

**Comment by ThePotatoGuy** [comment]
customizable pronouns is not planned to be implemented because of the significant changes that would have to be made. 

For example, here is a brainstorm I had a while back for customizable pronouns:
* screen with text fields for the pronoun text and radio butttons to determine if the pronoun is plural or singular.
* use a function to retrieve pronouns instead of directly using `[he]`, `[him]`, etc...

The only problem with this is the changes to every line that uses 1 of the 8 settable pronouns. It's a significant change and one that at the moment we can't justify.
